ofxclient/client.py

```diff
@@ -101,23 +133,19 @@ def post(self, query):
         garbage, path = splittype(i.url)
         host, selector = splithost(path)
         h = HTTPSConnection(host, timeout=60)
-        if host == 'ofx.discovercard.com':
-            # Discover requires a particular ordering of headers, so send the
-            # request step by step.
-            h.putrequest('POST', selector, skip_host=True,
-                         skip_accept_encoding=True)
-            h.putheader('Content-Type', 'application/x-ofx')
-            h.putheader('Host', host)
-            h.putheader('Content-Length', len(query))
-            h.putheader('Connection', 'Keep-Alive')
-            h.endheaders(query.encode())
-        else:
-            h.request('POST', selector, query,
-                      {
-                          "Content-type": "application/x-ofx",
-                          "Accept": "*/*, application/x-ofx",
-                          "User-Agent": "httpclient"
-                      })
+        # Discover requires a particular ordering of headers, so send the
+        # request step by step.
+        h.putrequest('POST', selector, skip_host=True,
+                     skip_accept_encoding=True)
+        h.putheader('Content-Type', 'application/x-ofx')
+        h.putheader('Host', host)
+        h.putheader('Content-Length', len(query))
+        h.putheader('Connection', 'Keep-Alive')
+        if self.accept:
```

Comment:
They're omitted for Discover; ``cli.py`` ``client_args_for_bank()`` sets them to False, so they'll be omitted here.